index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/0aca4b41e8.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>UI-Challenge</title>
+  </head>
+  <body>
+    <div id="ui-challenge">
+      <div class="container">
+        <img id="img1" src="/google-dots.png" alt="color_code" />
+        <i class="fa-solid fa-circle-chevron-left"></i>
+        <div class="header">
+          <img
+            src="https://res.cloudinary.com/teepublic/image/private/s--XuYxkWhu--/c_crop,x_10,y_10/c_fit,h_830/c_crop,g_north_west,h_1038,w_1038,x_-108,y_-104/l_upload:v1565806151:production:blanks:vdbwo35fw6qtflw9kezw/fl_layer_apply,g_north_west,x_-219,y_-215/b_rgb:000000/c_limit,f_jpg,h_630,q_90,w_630/v1569026033/production/designs/6035915_0.jpg"
+            alt="logo"
+          />
+          <h2>Aceagency</h2>
+        </div>
+
+        <h4 class="section-title">MENU</h4>
+        <div id="div2">
+          <ul>
+            <li><i class="fa-solid fa-briefcase"></i>My Offers</li>
+            <li><i class="fa-solid fa-clipboard"></i> User's Task</li>
+            <div class="highlight-menu-item">
+              <li>
+                <i class="fa-solid fa-tags"></i><span class="highlight-menu-text">Product Discounts</span>
+                <div class="highlight-message-count">2</div>
+              </li>
+            </div>
+            <li><i class="fa-solid fa-graduation-cap"></i>Case Study</li>
+            <li><i class="fa-solid fa-shield-halved"></i> Security</li>
+            <li><i class="fa-solid fa-bolt"></i>Your Ability</li>
+          </ul>
+        </div>
+
+        <div class="messages-section">
+          <h4 class="section-title">MESSAGES</h4>
+          <div class="navigate-icons"><i class="fa-solid fa-angle-left"></i> <i class="fa-solid fa-angle-right"></i></div>
+          <div id="div1">
+            <ul>
+              <li><i class="fa-solid fa-circle-user"></i>Max Maraston <img src="/icon-24.png" alt="num" /></li>
+              <li>
+                <i class="fa-solid fa-circle-user"></i>Celia W McCombs &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<i
+                  class="fa-solid fa-ellipsis"
+                ></i>
+              </li>
+              <li><i class="fa-solid fa-circle-user"></i>Edna J Critchlow <img src="/icon-1-24.png" alt="num1" /></li>
+              <li><i class="fa-solid fa-circle-user"></i>Dima Groshev</li>
+            </ul>
+
+            <button id="btn_msg">All Messages</button>
```

Comment:
Button type of 'menu' is invalid. You can avoid button type here since it can be controlled with JavaScript later.